<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cLightSourcesManager</class>
 <widget class="QWidget" name="cLightSourcesManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="MyCheckBox" name="checkBox_show_wireframe_lights">
     <property name="text">
      <string>Show wire-frame preview of lights</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_newLight">
       <property name="text">
        <string>New
light</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_duplicateLight">
       <property name="text">
        <string>Duplicate
light</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_deleteLight">
       <property name="text">
        <string>Delete
light</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_placeLight">
       <property name="text">
        <string>Place/move
light</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MyTabWidgetWithCheckboxes" name="tabWidget_lightSources">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTabWidgetWithCheckboxes</class>
   <extends>QTabWidget</extends>
   <header>my_tab_widget_with_checkboxes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyCheckBox</class>
   <extends>QCheckBox</extends>
   <header>my_check_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
